hardware_interface/include/hardware_interface/hardware_info.hpp

```diff
@@ -87,7 +87,7 @@ struct ActuatorInfo
   double offset = 0.0;
 };
 
-/// Contains semantic info about a given transmission loaded from XML (URDF)
+/// Contains semantic info about a given transmission loaded from URDF for a transmission
```

Comment:
It's kind a double...

```suggestion
/// Contains semantic info about a given transmission loaded from URDF
```